### ISSUE TITLE
webViewRender: Import statement may only appear at top level

### ISSUE THREAD

**Issue by ttraenkler** [inokawa/react-native-react-bridge#36]
I've bootstrapped a new react native project with expo managed workflow minimal typescript template and importing fine with

```ts
import {
	webViewRender,
	emit,
	useSubscribe,
} from "react-native-react-bridge/lib/web";
```

however, when invoking webViewRender

```ts
export default webViewRender(<div>a</div>);
```

transform fails with

```
Failed building JavaScript bundle.
WebApp.tsx: WebApp.tsx: Import statement may only appear at top level in file WebApp.tsx at 4:2
```

**Comment by inokawa** [comment]
Hi, @ttraenkler ,
It sounds similar to #18 so this may be a workaround.

Of course it should work with the original preset so I'll try to fix the problem.

**Comment by inokawa** [comment]
@ttraenkler 
Fixed in 0.6.0. Thank you for your reporting!

**Comment by ttraenkler** [comment]
Nice, that was quick - thanks a lot! 👍
